### PR TITLE
fix: Fix Google rechaptach

### LINKED ISSUE #276: BUGS: Fix Google rechaptach
Image

### PR DESCRIPTION
**Pull request by Rakibs511**
# Description

Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes #276 

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**:
* Firmware version:
* Hardware:
* Toolchain:
* SDK:

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: turbo.json [hunk 1]
```diff
@@ -33,7 +33,8 @@
         "NEXT_PUBLIC_BASE_URL",
         "NEXT_PUBLIC_SANITY_API_TOKEN",
         "SANITY_PREVIEW_SECRET",
-        "RECAPTCHA_SECRET_KEY"
+        "RECAPTCHA_SECRET_KEY",
+        "NEXT_PUBLIC_RECAPTCHA_SITEKEY"
```
Comment (MohiuddinRU) [inline turbo.json]: Site and Key are two different words. So, I think it should be like `NEXT_PUBLIC_RECAPTCHA_SITE_KEY`

**Review by TanvirAlam — DISMISSED** [review]
LGTM

**Review by TanvirAlam — APPROVED** [review]
LGTM

**Review by urmirahman — APPROVED** [review]
LGTM

**Pull request merged**